tensorflow_addons/losses/metric_learning.py

```diff
@@ -64,3 +64,27 @@ def pairwise_distance(feature, squared=False):
     )
     pairwise_distances = tf.math.multiply(pairwise_distances, mask_offdiagonals)
     return pairwise_distances
+
+
+@tf.function
+def angular_distance(feature):
```

Comment:
Could we annotate the type of `feature`?

Reply:
I was trying to adhere to the style of the above function, but I could certainly annotate the argument in the "feature: TensorLike" style if that is what you mean

Reply:
Please do :-) The type hint is not yet finish in our repo.